client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -6,29 +6,39 @@ const useStyles = makeStyles((theme) => ({
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    color: 'rgb(0,0,0,0.7)',
+    paddingLeft: '1px',
+    font: '16px Helvetica',
   },
   inputs: {
-    marginTop: '.8rem',
-    height: '2rem',
-    padding: '5px',
+    marginTop: '.3rem',
+    height: '2.5rem',
+    padding: '5px 5px 5px 15px',
+    border: '1px solid grey',
+    borderColor: 'lightgrey',
+    '&::placeholder': {
+      fontStyle: 'italic',
+    },
   },
   forgot: {
-    paddingRight: 10,
-    color: '#3a8dff',
+    fontSize: '11px',
+    fontWeight: 700,
+    color: 'black',
+    textDecoration: 'underline',
+    cursor: 'pointer',
+    '&:hover': {
+      opacity: '0.6',
+    },
   },
   submit: {
     margin: theme.spacing(3, 2, 2),
-    padding: 10,
-    width: 160,
-    height: 56,
-    borderRadius: theme.shape.borderRadius,
-    marginTop: 49,
-    fontSize: 16,
-    backgroundColor: '#3a8dff',
-    fontWeight: 'bold',
+    padding: '10px',
+    width: '160px',
+    height: '46px',
+    borderRadius: '1px',
+    marginTop: '49px',
+    backgroundColor: 'black',
+    font: 'lighter 12px Arial',
```

Comment:
curious why we would want to use a mix of Arial and Helvetica throughout the project?